cmd/multigateway/BUILD.bazel

```diff
@@ -0,0 +1,19 @@
+load("@rules_go//go:def.bzl", "go_binary", "go_library")
```

Comment:
we don't have to write this manually. These are generated automatically by bazel